linux/debian/rules

```diff
@@ -11,6 +11,12 @@ ifneq (ok,$(dpkg --compare-versions $(GOLANG_NATIVE_VERSION) ge 2:1.13 && echo o
 	export PATH := $(GODIR)/bin:$(PATH)
 endif
 
+GCC_NATIVE_VERSION = $(shell gcc -dumpversion 2>/dev/null || echo 0)
+ifneq (ok,$(shell dpkg --compare-versions $(GCC_NATIVE_VERSION) ge 8 && echo ok))
```

Comment:
That's a good catch! I am indifferent on what path to take regarding the golang hack, since it's only needed for Ubuntu/Bionic and we're going to be deprecating it in April anyways. Please feel free to either fix it in this PR or just ignore the problem it goes away in April.

Reply:
I'll... just ignore it then :eyes:

Reply:
Hm, actually this is downgrading go for all verisons of Ubuntu, no? We always enter the conditional. I'll file a ticket and a separate PR and we can discuss it there.